config/default.yml

```diff
@@ -855,6 +855,15 @@ PrePush:
     description: 'Run Test::Unit test suite'
     command: ['ruby', '-Ilib:test', '-rtest/unit', "-e 'exit! Test::Unit::AutoRunner.run'"]
 
+  Brakeman:
+    enabled: false
+    description: 'Check for security vulnerabilities'
+    required_executable: 'brakeman'
+    flags: ['--exit-on-warn', '--quiet', '--summary']
+    install_command: 'gem install brakeman'
+    include:
+      - '**/*.rb'
```

Comment:
Brakeman operates on more than just Ruby files. It also cares about ERB templates and other code.

It might be worth auditing their codebase to find all the file types and include an exhaustive list here, or we can omit the `include` option entirely and always run `Brakeman` on every push.